packages/app/components/search/search-status-bar.tsx

```diff
@@ -41,15 +43,17 @@ const SearchStatusBar = ({
         filterFields[field].schema = schema?.properties?.[field]
     })
 
-    const currentPrivileges = model.workflow
-        ? user.privilegesForModelAndWorkflowNode(
-              modelName,
-              model.workflow.currentNode
-          )
-        : []
+    const currentPrivileges = privilegesForModelAndWorkflowNode(
```

Comment:
The page throws an error if we don't have the model, so there's no reason to do a check for it.

Also moved "privilegesForModelAndWorkflowNode" off of the user and into a utility